includes/Renderer/Stories/Renderer.php

```diff
@@ -591,26 +591,14 @@ protected function render_story_with_poster() {
 			?>
 			<div class="web-stories-list__story-poster">
 				<a href="<?php echo esc_url( $story->get_url() ); ?>">
-					<?php
-					if ( $this->context->is_amp() ) {
-						// Set the dimensions to '0' so that we can handle image ratio/size by CSS per view type.
-						?>
-						<amp-img
-							src="<?php echo esc_url( $poster_url ); ?>"
-							layout="responsive"
-							width="0"
-							height="0"
-							alt="<?php echo esc_attr( $story->get_title() ); ?>"
-						>
-						</amp-img>
-	<?php } else { ?>
-						<img
-							src="<?php echo esc_url( $poster_url ); ?>"
-							alt="<?php echo esc_attr( $story->get_title() ); ?>"
-							width="<?php echo absint( $this->width ); ?>"
-							height="<?php echo absint( $this->height ); ?>"
-						>
-					<?php } ?>
+					<img
+						src="<?php echo esc_url( $poster_url ); ?>"
+						alt="<?php echo esc_attr( $story->get_title() ); ?>"
+						width="<?php echo absint( $this->width ); ?>"
+						height="<?php echo absint( $this->height ); ?>"
+						loading="lazy"
+						decoding="async"
```

Comment:
These changes to reasonable to me, but just want to confirm adding these attributes is the right thing. CC @swissspidy

Reply:
Yes. Without these it would be invalid AMP. I explicitly mentioned these in #10213 as well.

Reply:
Just confirming, no need to thumbs down 😢  . Should, implies some wiggle room, if it needs it, maybe you that wording.

Reply:
I swear this was meant to be a thumbs up